tox.ini

```diff
@@ -1206,6 +1206,7 @@ deps =
   docopt==0.6.2
   exceptiongroup==1.2.0
   flaky==3.7.0
+  fakeredis==2.23.3
```

Comment:
Do docker tests fail without that?

Reply:
The import will fail, maybe handle the input failure and skip the test?

Reply:
I have removed this one and seems the CI still could pass

Reply:
But the tests in docker-tests are like integration tests no?  I mean, we have a docker-compose.yml file which is deploying a real redis-cluster and seems in redis dockertests we are not mocking a redis